package.json

```diff
@@ -57,10 +58,9 @@
     "@strapi/strapi": "^4.0.0"
```

Comment:
Though it doesn't fix the linting issue you see here

Reply:
Thanks for your review @joshuaellis.
I'm able to continue with the migration now.

Just a note, as you saw, eslint threw an error due to `import/no-unresolved`.
This is actually a rule that comes from `eslint-config-airbnb`, which is a fairly popular eslint config set.

This seems to be because of the `package.json` exports field not being respected in that eslint rule.
And from the looks of it, they're not planning to support that. Source: https://github.com/airbnb/javascript/issues/2853

For now I've just added the `@strapi` packages to the ignore list of that eslint rule.
Though I still though you should know. Maby this'll pop up some more places down the line.

Reply:
Hmu in discord if you find any other issues and I'll try get back to you. RE Airbnb config, thanks for highlighting.